app/src/docs/AccountCard/AccountCardPageAdvanced.tsx

```diff
@@ -0,0 +1,216 @@
+/* @license Copyright 2023 @paritytech/polkadot-cloud authors & contributors
+SPDX-License-Identifier: GPL-3.0-only */
+
+import {
+  AccountCard,
+  ExtraComponentProps,
+  IconProps,
+} from "@packages/cloud-react/lib/recipes/AccountCard";
+import { Grid } from "@packages/cloud-react/lib/base/structure/Grid";
+import { Button } from "@packages/cloud-react/lib/buttons/Button";
+import { SimpleEditor } from "../lib/SimpleEditor";
+
+export const AccountCardPageAdvanced = () => {
+  const codeA = `
+import { AccountCard, IconProps, } from "@packages/cloud-react/lib/recipes/AccountCard";
+...
+const iconProps: IconProps = {
+  canCopy: true,
+  position: "right",
+  size: 3,
+  justify: "space-around",
+};
+...
+return (
+  <AccountCard icon={iconProps} address={"1f1yYj2bCFhJCTVdeWLDueUsrZynLAaj6jeMy18fjZ7Cr73"} ellipsis />
+);`;
+
+  const codeB = `
+import { AccountCard, IconProps, } from "@packages/cloud-react/lib/recipes/AccountCard";
```

Comment:
The code snippets need the production package path, so `@polkadot-cloud/react/...` instead of `@packages/cloud-react/lib/...`